daemons/upload-checksum-daemon/.gitignore

```diff
@@ -1,2 +1,6 @@
 .chalice/deployments/
 .chalice/venv/
+target/
+vendor/
+vendor.in/
```

Comment:
Why is vendor.in/ in the .gitignore? How would deployment in gitlab know about the wheels?

Reply:
oh that makes sense